### PR TITLE
108 resend member invitation

### PR DESCRIPTION
**Pull request by NitinCopods**
Resend Invitation in Member module developed

### CODE REVIEW

File: app/components/members/MemberListItem.tsx [hunk 1]
```diff
@@ -61,6 +72,15 @@ export default function MemberListItem({
               loggedInUser && 'cursor-not-allowed text-red-200'
             }`}
           />
+          <Icon
+            id="resend-member-invite"
+            tabIndex={0}
+            onClick={() => resendMail()}
```
Comment (yash6102000) [inline app/components/members/MemberListItem.tsx:78]: please add onkeyup event if not clickable by keyboard

File: app/components/DeletePopUp.tsx [hunk 2]
```diff
@@ -82,8 +82,8 @@ export default function DeletePopUp({
                     </div>
                     <div className="mt-3 text-center sm:mt-0 sm:ml-4 sm:text-left">
                       <p className="text-sm text-gray-500">
-                        {t('deletePopUp.alert')} {' '}
-                        {deleteItemType} '{deleteItem}'{subAlert}
+                        {t('deletePopUp.alert')} {deleteItemType} '{deleteItem}'
```
Comment (zia-copods) [inline app/components/DeletePopUp.tsx:85]: remove changes, if it's not from your side.

File: app/routes/members.tsx [hunk 3]
```diff
@@ -114,9 +114,26 @@ export const action: ActionFunction = async ({ request }) => {
           { status: 400 }
         )
       })
-
     return addHandle
   }
+  if (action.action === 'resend') {
```
Comment (zia-copods) [inline app/routes/members.tsx:119]: create a const file. For http methods

File: app/interface/Interface.tsx [hunk 4]
```diff
@@ -245,6 +245,12 @@ export enum QuestionTypes {
   text = 'TEXT',
 }
 
+export enum memberAction {
```
Comment (zia-copods) [inline app/interface/Interface.tsx:248]: Create a const file under `const` folder and add all the actions

**Review by zia-copods — CHANGES_REQUESTED** [review]

**Review by zia-copods — APPROVED** [review]
LGTM!

**Pull request merged**